RNZendeskChat.podspec

```diff
@@ -3,20 +3,20 @@ require 'json'
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 
 Pod::Spec.new do |s|
-  s.name         = "RNZendeskChat"
+  s.name         = 'RNZendeskChat'
   s.version      = package['version']
   s.summary      = package['description']
   s.license      = package['license']
   s.authors      = package['author']
   s.homepage     = package['homepage']
-  s.platform     = :ios, "10"
-  s.source       = { :git => "https://github.com/taskrabbit/react-native-zendesk-chat.git", :tag => "v#{s.version}" }
-  s.source_files = "ios/*.{h,m}"
+  s.platform     = :ios, '10'
+  s.source       = { git: 'https://github.com/taskrabbit/react-native-zendesk-chat.git', tag: "v#{s.version}" }
```

Comment:
Are you sure this is the correct syntax? I don't maintain many pod spec files -- so I wasn't sure if you entirely intended this change.

Reply:
(specifically commenting on the `:git =>` becoming `git: `

Reply:
@fbartho yes, it's correct syntax. 
https://stackoverflow.com/questions/44005410/need-to-refactor-to-the-new-ruby-1-9-hash-syntax/44005425